sdks/java/io/hcatalog/build.gradle

```diff
@@ -43,7 +43,7 @@ test {
 
 configurations.testRuntimeClasspath {
   resolutionStrategy {
-    def log4j_version = "2.8.2"
+    def log4j_version = "2.15.0"
```

Comment:
cc: @iemejia who specified an older version of SLF4J intentionally in https://github.com/apache/beam/commit/8a24eb8f3c65299a3f0dfb1bfa4a055ddc8dda5c#diff-63a43ac5cc26afc0e5cca037d586671d65fad2b34c2205e88cd46d8531910509

Reply:
Ping on this. Would this change cause any issues on 2.35.0 release?

Reply:
If we want to force the same version everywhere maybe it is better to remove the full ` configurations.testRuntimeClasspath` block (to avoid versions divergence in the future).

I had to explicitly pin an older version to avoid issues with the HCatalog tests if I remember correctly. If the full build passes with the updated version  I suppose it should be ok to go since log4j is mostly a transitive/runtime dependency.

Reply:
is it a 1.x thing getting upgraded to 2.x or is it a dep on something removed from 2.x?

Reply:
Answering my own question: no, the groupId/artifactId changed, so my PR doesn't upgrade 1.x to 2.15.0